Reply/app/src/main/java/com/materialstudies/reply/util/FragmentExtensions.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.materialstudies.reply.util
+
+import androidx.fragment.app.Fragment
+
+/**
+ * Sets the exit and reenter transitions, or nulls them out if not provided.
+ */
+fun Fragment.setOutgoingTransitions(
+        exitTransition: Any? = null,
```

Comment:
Can these types be `Transition?` instead of `Any`?

Reply:
the Fragment setters are typed as `Any` (Object), because they support androidx and framework, and i think it's best to keep the same signature for a wrapper method

Reply:
Oh cool. Makes sense